test/actions/katello/content_view_version_test.rb

```diff
@@ -28,98 +28,6 @@ class IncrementalUpdateTest < TestBase
       katello_repositories(:fedora_17_x86_64)
     end
 
-    it 'plans' do
```

Comment:
We kept this test in master but changed it up a little. Will matching it up with master work? https://github.com/Katello/katello/blob/master/test/actions/katello/content_view_version_test.rb#L31